src/hostapi/pulseaudio/pa_linux_pulseaudio.c

```diff
@@ -952,6 +952,22 @@ PaError OpenStream( struct PaUtilHostApiRepresentation *hostApi,
         goto error;
     }
 
+    /* Allocate memory for source and sink names. */
+    char defaultSourceStreamName[] = "Portaudio source";
+    char defaultSinkStreamName[] = "Portaudio sink";
+
+    stream->sourceStreamName = (char*)PaUtil_AllocateMemory(sizeof(defaultSourceStreamName));
```

Comment:
These should be +1 for NULL at the end

Reply:
sizeof() already includes the null character by virtue of the string being a constant array. Agree?

Reply:
Ok I made small code like:
```
#include <stdio.h>
#include <string.h>

int main(){
   const char test1[] = "test";
   const char *test2 = "test";
   printf("Sizeof: %d strlen %d\n", sizeof(test1), strlen(test1));
   printf("Sizeof: %d strlen %d\n", sizeof(test2), strlen(test2));

return 1;
}
```
And sizeof gives correct answer and strlen gives string len so no need for +1. Btw I was wrong if used const char * then sizeof test is 8 not 5 as above gives.

Reply:
That's expected as it's measuring the size of a pointer (which is 4 or 8 bytes depending on whether the system is 32 or 64 bit). Thus, you'd always want to use strlen() for those.